web/src/inbox_ui.ts

```diff
@@ -1740,6 +1964,64 @@ export function update(): void {
         }
     }
 
+    for (const [folder_id, folder_info] of folders_info.entries()) {
+        const folder_dict = channel_folders_dict.get(folder_id);
+        let name: string;
+        if (folder_id === PINNED_CHANNEL_FOLDER_ID) {
+            name = $t({defaultMessage: "PINNED CHANNELS"});
+        } else if (folder_id === OTHER_CHANNELS_FOLDER_ID) {
+            name = $t({defaultMessage: "OTHER CHANNELS"});
+        } else {
+            name = channel_folders.get_channel_folder_name_from_id(folder_id);
+        }
```

Comment:
I feel like a function to do this would be useful both here and in the loop for https://github.com/zulip/zulip/pull/35152/commits/7b8a6ef27b5a1145322670f3d3ac901df6a6367a#r2196185448.

Reply:
Yeah, done.